src/utilities.js

```diff
@@ -47,17 +48,22 @@ export function deleteLotFromPortfolio(id) {
 // TODO: need to convert this to sum values
 export const getDataForChart = (ticker) => {
   const pricesJson = Constants.API_PRICES[ticker];
-  const xAxisLabels = [];
-  const yAxisLabels = [];
-  for (let i = 0; i < pricesJson.length; i++) {
-    xAxisLabels.push(pricesJson[i].date.split('T')[0]);
-    yAxisLabels.push(pricesJson[i].close);
-  }
+  if (pricesJson) {
```

Comment:
Some people prefer a short-circuit style here. It's a matter of taste.

`if (!pricesJson) return;`

and then you don't need to have `if (pricesJson)` section indented.

Reply:
done